FrontEnd/src/Appointment/AppointmentPage.css

```diff
@@ -1,3 +1,161 @@
-.appointmentPage{
-    margin-top: 120px;
+#root {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
 }
+.appointmentPage {
+  margin-top: 100px;
+  background-color: var(--backgroundColor);
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  align-items: center;
+}
+.appointments{
+  width: 100%;
+  max-width: 81rem;
+  padding: 0 1rem;
+}
+
+h2, p {
+  color: var(--darkTwo);
+}
+
+.appointmentTab {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--darkOne);
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  padding: 1.5rem;
+  margin: 1rem 0;
+  border-radius: 8px;
+  color: var(--lightOne);
+  width: 100%;
+  max-width: 81rem;
+}
+
+.appointmentInfo {
+  flex: 1;
+}
+
+.appointmentUser {
+  font-size: 1.2rem;
+  font-weight: bold;
+  margin: 0;
+  color: var(--lightOne);
+}
+
+.appointmentTitle {
+  font-size: 1rem;
+  margin: 0.5rem 0;
+}
+
+.appointmentDetail {
+  display: flex;
+  align-items: center;
+  font-size: 0.9rem;
+  margin: 0.2rem 0;
+  color: var(--lightOne)
+}
+
+.appointmentDetail i {
+  margin-right: 0.5rem;
+}
+
+.appointmentActions {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.appointmentActions button {
+  margin-top: 0.5rem;
+  background-color: var(--color6);
+  color: #000;
+  border: none;
+  padding: 0.5rem 1rem;
+  border-radius: 4px;
+  cursor: pointer;
+  font-size: 0.9rem;
+  width: 200px;
+}
+
+.appointmentActions button:hover {
+  background-color: #e0e0e0;
+}
+
+@media (max-width: 1300px) {
+  .appointments, .appointmentTab {
+    max-width: 70rem;
+  }
+}
+
+@media (max-width: 1100px) {
+  .appointments, .appointmentTab {
+    max-width: 60rem;
+  }
+}
+
+@media (max-width: 900px) {
+  .appointments, .appointmentTab {
+    max-width: 50rem;
+  }
+}
+
+@media (max-width: 700px) {
+  .appointments, .appointmentTab {
+    max-width: 40rem;
+  }
+}
+
+@media (max-width: 500px) {
+  .appointments, .appointmentTab {
+    max-width: 30rem;
+    padding: 1rem; /* Reduce padding for smaller screens */
+  }
+
+  .appointmentTab {
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .appointmentActions {
+    align-items: flex-start;
+    margin-top: 1rem;
+  }
+
+  .appointmentActions button {
+    width: 100%;
+  }
+}
+
+
+/* .appointmentTab {
```

Comment:
Are these commented out for later use?

Reply:
This was from a stashed commit I used for saving my work when my laptop was acting up. While backtracking on working commits, I must have selected it. It's been updated in my new PR though.